ArkKit/ark-render-kit/GameStateRenderer.swift

```diff
@@ -6,5 +8,8 @@
  * like `SwiftUI`.
  */
 protocol GameStateRenderer: AnyObject {
+    typealias ScreenResizeDelegate = (CGSize) -> Void
+    
     func render(canvas: Canvas, with canvasContext: CanvasContext)
+    func onScreenResize(_ delegate: @escaping ScreenResizeDelegate)
```

Comment:
Am wondering if we should move `onScreenResize` to the `AbstractView` instead - since screen resizes are propagated from the view and not the role of the renderer.